tests/unit/test_events.py

```diff
@@ -1500,6 +1506,92 @@ def test_parse_check_rerun_release(self, check_rerun):
         assert event_object.tests_targets_override == {"fedora-rawhide-x86_64"}
         assert event_object.actor == "lbarcziova"
 
+    @pytest.mark.parametrize(
+        "upstream_project_url, upstream_tag_template, pre_check_result, "
+        "tag_name, repo_namespace, repo_name",
+        [
+            (
+                "https://github.com/redis-namespace/redis",
+                None,
+                True,
+                "7.0.3",
+                "redis-namespace",
+                "redis",
+            ),
+            (
+                "https://github.com/redis-namespace/redis",
+                "no-version-tag",
+                True,
+                "no-version-tag",
+                "redis-namespace",
+                "redis",
+            ),
+            (
+                "https://github.com/redis-namespace/redis",
+                "v{version}",
+                True,
+                "v7.0.3",
```

Comment:
okay, this probably answers my question from above :D

although I'm contemplating if the tag name is already present in the message event and if it's, we should probably use it

Reply:
no, it is not in the message (there is an example message in the test data). We discussed with @lachmanfrantisek  that if there is `upstream_tag_template` defined in config, we will use it to get the tag and if it is not, take the version as the tag.

Reply:
understood: you solved it well